framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -1025,8 +1026,21 @@ private void processPostconditionsAndConditionalPostconditions(
                     thenStore.insertValue(r, anno);
                 }
             } catch (FlowExpressionParseException e) {
+                Result result;
+                if (e.isFlowParseError()) {
+                    Object[] args = new Object[e.args.length + 1];
+                    args[0] = ElementUtils.getVerboseName(TreeUtils.elementFromUse(n.getTree()));
+                    for (int i = 0; i < e.args.length; i++) {
```

Comment:
You might want to use https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#arraycopy-java.lang.Object-int-java.lang.Object-int-int- instead, to make explicit and shorter that you're just copying things around.
(There currently is always just one argument and the error message expects exactly 2 parameters, so maybe this generality is overkill...)